apps/web/src/app/[locale]/(dashboard)/[wsId]/tasks/boards/enhanced-boards-view.tsx

```
@@ -0,0 +1,2328 @@
+              </div>
+            ))}
+          </div>
+        </div>
+      </div>
+
+      {/* Gantt Chart - Fixed Height Container */}
+      <div 
+        className="border rounded-lg bg-background relative"
+        style={{ height: '320px' }} // Fixed height instead of expanding
+      >
+        <style dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)